src/tests/ftest/container/container_async.py

```diff
@@ -57,52 +47,26 @@ def cb_func(event):
     GLOB_RC = event.event.ev_error
     GLOB_SIGNAL.set()
 
-class ContainerAsync(Test):
+class ContainerAsync(TestWithServers):
     """
     Tests DAOS pool connect permissions (non existing pool handle, bad uuid)
     and close.
+
+    :avocado: recursive
     """
 
     def __init__(self, *args, **kwargs):
         super(ContainerAsync, self).__init__(*args, **kwargs)
         self.container1 = None
         self.container2 = None
 
-    def setUp(self):
-        self.agent_sessions = None
-        self.hostlist = None
-        self.pool = None
-
-        # get paths from the build_vars generated by build
-        with open('../../../.build_vars.json') as build_file:
-            build_paths = json.load(build_file)
-        self.basepath = os.path.normpath(build_paths['PREFIX'] + "/../")
-
-        self.server_group = self.params.get("server_group", '/server/',
-                                            'daos_server')
-
-        # setup the DAOS python API
-        self.context = DaosContext(build_paths['PREFIX'] + '/lib/')
-        self.pool = None
-
-        self.hostlist = self.params.get("test_machines", '/run/hosts/*')
-        self.hostfile = write_host_file.write_host_file(self.hostlist,
-                                                        self.workdir)
-        print("Host file is: {}".format(self.hostfile))
-
-        self.agent_sessions = AgentUtils.run_agent(self.basepath, self.hostlist)
-        server_utils.run_server(self.hostfile, self.server_group, self.basepath)
-        time.sleep(10)
-
     def tearDown(self):
         try:
             if self.pool is not None and self.pool.attached:
```

Comment:
Why is this necessary?

Reply:
I'm not sure what you are referring to.  The line of code you are commenting on is untouched by the patch.